The-SMIRNOFF-force-field-format.md

```diff
@@ -231,21 +231,20 @@ In keeping with the AMBER force field philosophy, especially as implemented in s
 
 Here is an example:
 ```XML
-<ChargeIncrementModel version="0.3" number_of_conformers="10" quantum_chemical_method="AM1" partial_charge_method="CM2">
+<ChargeIncrementModel version="0.3" number_of_conformers="1" partial_charge_method="AM1-Mulliken">
```

Comment:
I still think we may want to separate this into two tags, since even AM1-Mulliken methods may expose options such as
* Whether to use multiple conformers (e.g. AM1 vs ELF AM1)
* How many conformers to generate (e.g. ELF vs ELF10)
* What tolerance to use for minimization
We may not want to cram tons of additional attributes into this single tag to keep qualifying the 

Also, the `number_of_conformers` attribute will no longer be relevant when we move to graph convolutional networks. Instead, it may be a model name.

Reply:
Just to record what we discussed earlier this week -- My thought is that there are so many different ways this tag could go, that I'm just going to make it a string `partial_charge_method` and an int `number_of_conformers` in the initial version. This will almost certainly change in the future, but we can modify the spec once we have a clearer idea of the scope.